R/golem_utils_ui.R

```diff
@@ -8,7 +8,6 @@
 #'
 #' @examples
 #' list_to_li(c("a", "b"))
-#' @importFrom shiny tags tagAppendAttributes tagList
```

Comment:
All changes in this file are just to remove the unneeded importing. This file is one generated by golem on demand, but since shiny is imported as a whole, these imports are not needed.